plugin/src/main/kotlin/com/github/triplet/gradle/play/PlayPublisherPlugin.kt

```diff
@@ -175,7 +175,9 @@ class PlayPublisherPlugin : Plugin<Project> {
                 project.tasks.findByName(
                         (variant as InstallableVariantImpl).variantData.getTaskName("bundle", ""))
                         ?.let { dependsOn(it) }
-                        ?: logger.warn("Bundle task not found. Publishing App Bundles may not work.")
+                        ?: logger.warn("Bundle task not found, make sure to use " +
+                                               "'com.android.tools.build:gradle:3.2.+'. " +
```

Comment:
Any chance we can change this to not suggest wildcard versioning?

Reply:
Yeah, I was trying to say "use 3.2+" but it looks like I'm telling devs to add that dependency. 👻